### PR TITLE
convert redhat_packages.* facts to raw tasks. Parse ssh output. Closes#117

### PR DESCRIPTION
**Pull request by chambridge**

### CODE REVIEW

File: library/run_cmds.py [hunk 1]
```diff
@@ -51,9 +51,7 @@
 # passed in through the playbook (attached
 # to the field requested using an '_').
 
-DEFAULT_CMD_DICT = {"SysId": file_commands.SystemIdRhoCmd,
```
Comment (noahl) [inline library/run_cmds.py]: Can we just remove `DEFAULT_CMD_DICT`? :)

File: library/spit_results.py [hunk 2]
```diff
@@ -39,10 +199,17 @@ def write_to_csv(self):
             file_size = os.path.getsize(f_path)
             vals = self.vals
             fields = sorted(vals[0].keys())
+            try:
+                fields.remove('systemid.contents')
```
Comment (noahl) [inline library/spit_results.py]: There might be a problem if `redhat-packages.results` is in fields but `systemid.contents` is not.

I'll send a PR today which will turn fields into a set, which should make this much easier to solve. Without that I don't really see an elegant way to do it.

File: library/run_cmds.py (hunk 1 above)
Comment (chambridge) [reply library/run_cmds.py]: This will be handled by the remaining work in #68.

File: library/spit_results.py (hunk 2 above)
Comment (chambridge) [reply library/spit_results.py]: I've pushed a commit to separate these two removals. I've tried to only do minimal changes here base on the direction you mentioned on Friday. Looking forward to the PR.

**Pull request merged**